CHANGELOG.md

```diff
@@ -121,6 +121,8 @@ release.
 - BREAKING: Remove the zero bucket boundary from `http.server.request.duration`
   and `http.client.request.duration`.
   ([#318](https://github.com/open-telemetry/semantic-conventions/pull/318))
+- BREAKING: Remove `server.socket.address` from HTTP and RPC client metrics.
```

Comment:
Can this be represented in the schema file?

Reply:
I don't think so. There's https://github.com/open-telemetry/opentelemetry-specification/issues/3288 but that's for the inverse (adding a metric attribute as opposed to removing a metric attribute).